backend/routes_cycle_count.py

```
@@ -388,14 +388,42 @@ def update_item_from_count_result(cycle_count_item, count_result):
+            return jsonify({'error': 'Authentication required', 'reason': 'No token provided'}), 401
+
+        token = auth_header.split(' ')[1]
+
+        try:
+            import jwt
+            from flask import current_app, g
+            # Decode JWT token
+            payload = jwt.decode(token, current_app.config['SECRET_KEY'], algorithms=['HS256'])
+
+            # Get user from database to ensure they still exist and are active
+            user = User.query.get(payload['user_id'])
```

Ruff (0.11.9)

Undefined name `User`

(F821)